src/main/java/com/cs56fitnessapp/models/Day.java

```diff
@@ -1,13 +1,55 @@
 package com.cs56fitnessapp.models;
 
 /**
- * @author Ksenia Koldaeva
- * Created: 9/17/17
- * Last Updated: 10/1/17
+ * @author Saori Shigehisa
+ * Created: 10/6/17
+ * Last Updated: 10/14/17
  */
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Date;
+
 
 public class Day implements ActivityFacts, DietFacts {
 
+    /** date */
+    private LocalDate date;
+
+    /** User */
+    private User user;
+
+    /** calorieGoal */
+    private double calorieGoal;
+
+    /** WorkoutList */
+    private ArrayList < Workout > workoutList;
+
+    /** MealList */
+    private ArrayList < Meal > mealList;
+
+    public Day (LocalDate date, User user, double caloriesGoal, ArrayList < Workout > workoutList, ArrayList < Meal > mealList){
+      this.date = date;
+      this.user = user;
+      this.calorieGoal = calorieGoal;
+      this.workoutList = workoutList;
+      this.mealList = mealList;
+    }
+
+
+
+    public boolean isToday() {
+        LocalDate now = LocalDate.now();
+
+        if (now.getYear() == date.getYear() && now.getMonth() == date.getMonth() && now.getDayOfMonth() == date.getDayOfMonth()) {
+            return true;
+        } else {
+            return false;
+
+        }
+    }
+
+    //Kcal ~= METS * bodyMassKg * timePerformingHours
+    public double getCaloriesLeft() { return 0.0; }
 
     /* ActivityFacts Interface */
     @Override
```

Comment:
Will need to implement overridden methods so they return actual values, not default 0.

Reply:
From which interface or class should I override for getCaloriesLeft?

Reply:
Let's think about it this way:
I am coming to you (you have the Day object in your hands), and I am asking - how many calories I have eaten so far? 
You should know the answer to this question, because you keep track of my meals. E.g. you know that I had breakfast, lunch and snack. You also know how much each of them worth because according to the UML (our API) you can call `getCaloriesIn()` on each object of  type `Meal`. So you calculate the sum of calories inputs that I had and return me the result. 

* This is what overridden method `.getCaloriesIn()` in your class should return. Currently it gives me 0 all the times, no matter how many meal I actually had during the day - my calculator always shows me, that I  have eaten 0 calories. I eat more, it's still 0, and probably I will not lose weight with this logic.

* Method `.getCaloriesLeft()` is specific to your class. You do not override it, instead you create it, and knowing 3 values - how many calories I have eaten, how many calories I have burned and what is my daily calorie goal, you give me back the value of the calories that I have left to spend.

* You should use other classes APIs (User, Meal, Workou) to implement these methods. `Meal` is not ready yet. Wait for it. Other classes such as `User` and `Workout` you can use.